### PR TITLE
Rename Project Firefly to App Builder

### PR DESCRIPTION
**Pull request by icaraps**
Update text references only (links are untouched)

### CODE REVIEW

File: src/pages/getting_started/common_troubleshooting.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ Here are troubleshooting guides for some of the most common issues as you develo
...
 ## Before you proceed
 
-- Check your Node version and tool versions to ensure they are supported by Project Firefly and up-to-date. You can find the latest supported version [here](index.md).
+- Check your Node version and tool versions to ensure they are supported by App Builder and up-to-date. You can find the latest supported version [here](index.md).
 - Check if your application is on Dropbox or OneDrive as file watchers sometimes cause unexpected errors. 
 
 ## General debugging
 
-When your action code doesn't work as expected, you may want to investigate into what exactly went wrong. Project Firefly provides the [Logging SDK](https://github.com/adobe/aio-lib-core-logging), please check out [Firefly's Application Logging](../guides/application_logging.md) for more details. 
+When your action code doesn't work as expected, you may want to investigate into what exactly went wrong. App Builder provides the [Logging SDK](https://github.com/adobe/aio-lib-core-logging), please check out [Firefly's Application Logging](../guides/application_logging.md) for more details.
```
Comment (shazron) [inline src/pages/getting_started/common_troubleshooting.md:20]: missed one reference here

File: src/pages/resources/barcode-reader/test.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ keywords:
 
 # Lesson 3: Testing a Serverless Action
 
-A Firefly App created with `aio app init` has a [jest](https://jestjs.io/) test structure by default. We'll show how to take advantage of it.
+A App Builder App created with `aio app init` has a [jest](https://jestjs.io/) test structure by default. We'll show how to take advantage of it.
```
Comment (shazron) [inline src/pages/resources/barcode-reader/test.md:11]: An App Builder App

File: src/pages/resources/customer-dashboard/lesson5.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ Verify that the new action is working by running the app locally with `aio app r
 
 ![generate-code](assets/generate-code.png)
 
-*Note: Visit the codelab [Headless Apps with Project Firefly](../barcode-reader/index.md) to learn more about building a headless app for barcode generation.*  
+*Note: Visit the codelab [Headless Apps with App Builder](../barcode-reader/index.md) to learn more about building a headless app for barcode generation.*  
 
 Now that you have it set up in Firefly app, next step is to create a marketing workflow in Campaign Standard which takes care of receiving external signals from the app and sending promotion emails. To do that, go to *Marketing Activities > Create > Workflow*. Define the properties of your workflow, and finish the creation.
```
Comment (shazron) [inline src/pages/resources/customer-dashboard/lesson5.md:80]: missed this reference

File: src/pages/resources/debugging/lesson3.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ contributors:
 
 # Lesson 3: Managing Application Logs
 
-Logging is a great add-on for debugging apps during development as well as at runtime. Project Firefly [SDK](https://github.com/adobe/aio-sdk) and [CLI](https://github.com/adobe/aio-cli) provide commodities for developers to log and retrieve information and data from their apps.  
+Logging is a great add-on for debugging apps during development as well as at runtime. App Builder [SDK](https://github.com/adobe/aio-sdk) and [CLI](https://github.com/adobe/aio-cli) provide commodities for developers to log and retrieve information and data from their apps.  
 
 In our sample app, `logger` is used throughout the `hello` action. It is provided by the [Firefly's core Logging library](https://github.com/adobe/aio-lib-core-logging).
```
Comment (shazron) [inline src/pages/resources/debugging/lesson3.md:15]: missed this reference

File: src/pages/resources/event-driven/lesson1.md [hunk 5]
```diff
@@ -33,18 +33,18 @@ downloaded automatically, you will need the private key to generate JWT token.
 
 ## Initialize a Firefly app using CLI template
```
Comment (shazron) [inline src/pages/resources/event-driven/lesson1.md:34]: missed this reference

File: src/pages/resources/event-driven/lesson1.md [hunk 6]
```diff
@@ -33,18 +33,18 @@ downloaded automatically, you will need the private key to generate JWT token.
...
 
 You will be presented with a few options, first question is "Which Adobe I/O App features do you want to enable for this project?" In this lab, we keep all of them. The second question is "Which type of sample actions do you want to create?" in this lab, we will select `Generic`, please remember to keep the `publish-event` template.
-Name your action, then you have created your project firefly template. Now you can use this template to start your app
+Name your action, then you have created your App Builder template. Now you can use this template to start your app
 
 ![event-provider](assets/publish-event-cli.png)
 
-In this lab, user will create a webpage using this generic template. if this is your first time to use project firefly, follow below instructions:
+In this lab, user will create a webpage using this generic template. if this is your first time to use App Builder, follow below instructions:
 * [Creating your First Firefly App](../../getting_started/first_app.md)
```
Comment (shazron) [inline src/pages/resources/event-driven/lesson1.md:48]: missed this reference

File: src/pages/resources/event-driven/welldone.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ In addition, more to read about Adobe I/O Event:
 
 You have learned: 
 
-* How to start a project firefly app 
+* How to start a App Builder app 
 * Register event provider using custom event
 * Testing the flow by emitting event from firefly app
```
Comment (shazron) [inline src/pages/resources/event-driven/welldone.md:31]: missed this reference

File: src/pages/resources/journaling-events/lesson1.md [hunk 8]
```diff
@@ -11,12 +11,12 @@ contributors:
 # Lesson 1: Create an Event Provider using Firefly
```
Comment (shazron) [inline src/pages/resources/journaling-events/lesson1.md:11]: missed this reference

File: src/pages/resources/journaling-events/lesson1.md [hunk 9]
```diff
@@ -11,12 +11,12 @@ contributors:
...
 In this lesson, we will do the follow steps:
-- Create an event provider using Project Firefly template
+- Create an event provider using App Builder template
 - Register the App as event provider 
 - Fire Events
 - Scheduling cron jobs with alarms
 
-## Create an event provider using Project Firefly template
+## Create an event provider using App Builder template
 In this codelab, to provide an end to end solution, we need to have an event provider generating tons of events sending to journaling API, and the events provider need to be configurable to send x events/min to help user to test. so we choose to use OpenWhisk Alarms Package in Firefly application to create
```
Comment (shazron) [inline src/pages/resources/journaling-events/lesson1.md:20]: missed this reference "use OpenWhisk Alarms Package in Firefly application"

File: src/pages/resources/spectrum-intro/lesson4.md [hunk 10]
```diff
@@ -6,7 +6,7 @@ keywords:
   - Developer Tooling
 ---
 
-# Lesson 4: React Spectrum in Project Firefly
+# Lesson 4: React Spectrum in App Builder
 
 To initialize a Firefly app, let’s use the init command from the [CLI](https://github.com/adobe/aio-cli).
```
Comment (shazron) [inline src/pages/resources/spectrum-intro/lesson4.md:11]: missed this reference

File: src/pages/resources/transclusions/_requirements.md [hunk 11]
```diff
@@ -11,6 +11,6 @@ This codelab is useful for anyone who are getting started with developing Firefl
 
 If not already done, you should follow the below links to have your environment set up before starting the codelab:
 
-* [How to Get Access to Project Firefly](../../overview/getting_access.md)
+* [How to Get Access to App Builder](../../overview/getting_access.md)
 * [Setting up Your Environment](../../getting_started/index.md)
 * [Creating your First Firefly App](../../getting_started/first_app.md)
```
Comment (shazron) [inline src/pages/resources/transclusions/_requirements.md:16]: missed this reference

File: src/pages/resources/videos/developers-live/deep-dive.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Adobe Developers Live | Project Firefly Deep Dive
+title: Adobe Developers Live | App Builder Deep Dive
 description: Join us and see Firefly in action - code, security, headfull and headless apps, CI/CD and more.
```
Comment (shazron) [inline src/pages/resources/videos/developers-live/deep-dive.md:3]: missed this reference

File: src/pages/resources/videos/developers-live/deep-dive.md [hunk 13]
```diff
@@ -8,7 +8,7 @@ keywords:
   - Developer Tooling  
 ---
 
-# Adobe Developers Live | Project Firefly Deep Dive
+# Adobe Developers Live | App Builder Deep Dive
 
 Join us and see Firefly in action: code, security, headfull and headless apps, CI/CD and more.
```
Comment (shazron) [inline src/pages/resources/videos/developers-live/deep-dive.md:13]: missed this reference

File: src/pages/resources/videos/exploring/projects-and-workspaces.md [hunk 14]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Exploring Project Firefly - Projects and Workspaces
-description: The Project Firefly team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.  
+title: Exploring App Builder - Projects and Workspaces
+description: The App Builder team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.
```
Comment (shazron) [inline src/pages/resources/videos/exploring/projects-and-workspaces.md:3]: missed this reference "to help your team collaborate on building Firefly Applications."

File: src/pages/resources/videos/exploring/projects-and-workspaces.md [hunk 15]
```diff
@@ -1,16 +1,16 @@
...
   - Extensibility
   - API Documentation
   - Developer Tooling  
 ---
 
-# Exploring Project Firefly - Projects and Workspaces
+# Exploring App Builder - Projects and Workspaces
 
-The Project Firefly team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.
+The App Builder team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.
```
Comment (shazron) [inline src/pages/resources/videos/exploring/projects-and-workspaces.md:13]: missed this reference "to help your team collaborate on building Firefly Applications."

File: src/pages/resources/videos/overview/architecture.md [hunk 16]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Project Firefly - A Breakdown of Firefly's Architecture
+title: App Builder - A Breakdown of Firefly's Architecture
```
Comment (shazron) [inline src/pages/resources/videos/overview/architecture.md:2]: missed this reference "Firefly's Architecture"

**Review by shazron — CHANGES_REQUESTED** [review]

**Comment by icaraps** [comment]
Ping @sarahxxu

**Comment by icaraps** [comment]
Thx @shazron for the review ! I just committed another batch of renames

**Review by shazron — APPROVED** [review]

File: src/pages/resources/barcode-reader/test.md (hunk 2 above)
Comment (shazron) [reply src/pages/resources/barcode-reader/test.md:11]: @icaraps I think you missed this one

**Comment by icaraps** [comment]
@sarahxxu good to merge ?

**Review by shazron — APPROVED** [review]

**Pull request merged**